scripts/spack/packages/umpire/package.py

```diff
@@ -78,10 +78,15 @@ class Umpire(CachedCMakePackage, CudaPackage, ROCmPackage):
 
     depends_on('cmake@3.8:', type='build')
     depends_on('cmake@3.9:', when='+cuda', type='build')
-    depends_on('cmake@:3.20', when='+rocm', type='build')
+    # BEGIN AXOM EDIT
+    # Prevents spack spec with CMake 3.21
+    #depends_on('cmake@:3.20', when='+rocm', type='build')
+    # END AXOM EDIT
     depends_on('cmake@3.14:', when='@2022.03.0:')
 
-    depends_on('blt@0.5.0:', type='build', when='@2022.03.0:')
+    # BEGIN AXOM EDIT
+    depends_on('blt@0.5.1:', type='build', when='@2022.03.0:')
```

Comment:
We need to remember to push these two changes back up to @davidbeckingsale 's branch, including the other related packages as well.